perf/perf_hv_gpci.py

```diff
@@ -59,14 +59,27 @@ def setUp(self):
                 self.cancel('%s is needed for the test to be run' % package)
 
         # Collect all hv_gpci events
-        self.list_of_hv_gpci_events = []
+        self.list_phys = []
+        self.list_sibling = []
+        self.list_partition = []
+        self.list_hw = []
+        self.list_noid = []
```

Comment:
If there are multiple empty list  how about 
`self.list_phys, self.list_sibling, self.list_partition, self.list_hw, self.list_noid = ([],)*5`

Reply:
test is not working as expected with suggested change